service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -51,179 +43,123 @@
+            Element<HearingOrder> additionalOrder = element(orderTwoId, cmo);
+            Element<HearingBooking> hearingToBeRemoved = element(hearingOneId, hearing(orderToBeRemovedId));
+            Element<HearingBooking> additionalHearing = element(hearingTwoId, hearing(orderTwoId));
 
-            CaseDetails caseDetails = asCaseDetails(CaseData.builder()
-                .hearingDetails(newArrayList(hearing1))
-                .draftUploadedCMOs(newArrayList(agreedCMO))
-                .id(12345678901234566L)
-                .build());
+            List<Element<HearingOrder>> draftCaseManagementOrders = newArrayList(
+                orderToBeRemoved,
+                additionalOrder);
 
-            caseDetails.getData().put("migrationId", migrationId);
+            List<Element<HearingBooking>> hearingBookings = newArrayList(hearingToBeRemoved, additionalHearing);
+
+            CaseDetails caseDetails = caseDetails(migrationId, incorrectFamilyManNumber, draftCaseManagementOrders,
+                hearingBookings);
 
             CaseData extractedCaseData = extractCaseData(postAboutToSubmitEvent(caseDetails));
 
-            assertThat(extractedCaseData.getDraftUploadedCMOs()).containsOnly(agreedCMO);
-            assertThat(extractedCaseData.getHearingOrdersBundlesDrafts()).isNull();
+            assertThat(extractedCaseData.getDraftUploadedCMOs()).isEqualTo(draftCaseManagementOrders);
+            assertThat(extractedCaseData.getHearingDetails()).isEqualTo(hearingBookings);
         }
 
         @Test
-        void shouldNotMigrateToHearingBundlesWhenMigrationIdIsNotExpected() {
+        void shouldThrowAnExceptionIfCaseDoesNotContainDraftCaseManagementOrders() {
+            List<Element<HearingBooking>> hearingBookings = newArrayList(newArrayList());
 
-            Element<HearingBooking> hearing1 = element(buildHearing(now().plusDays(1), cmoId1));
+            CaseDetails caseDetails = caseDetails(migrationId, familyManNumber, null,
+                hearingBookings);
 
-            Element<HearingOrder> agreedCMO1 = buildCMO(cmoId1, hearing1.getValue().toLabel(), SEND_TO_JUDGE);
+            assertThatThrownBy(() -> postAboutToSubmitEvent(caseDetails))
+                .getRootCause()
+                .hasMessage("No draft case management orders in the case");
+        }
 
+        private CaseDetails caseDetails(String migrationId,
+                                        String familyManNumber,
+                                        List<Element<HearingOrder>> draftCaseManagementOrders,
+                                        List<Element<HearingBooking>> hearingBookings) {
             CaseDetails caseDetails = asCaseDetails(CaseData.builder()
-                .hearingDetails(newArrayList(hearing1))
-                .draftUploadedCMOs(newArrayList(agreedCMO1))
-                .id(caseId1)
+                .familyManCaseNumber(familyManNumber)
+                .draftUploadedCMOs(draftCaseManagementOrders)
```

Comment:
Are the draft cmos not saved in `hearingOrdersBundlesDrafts`?

Reply:
Screen attatched to the ticket looks like the hearing is stored in `draftUploadedCMOs` rather than `hearingOrdersBundlesDrafts` no? Looks like an old case to me.